<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrowseSimulations</class>
 <widget class="QDialog" name="BrowseSimulations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Browse Simulations</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Simulation:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ListViewChange" name="listViewSimulation"/>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Filter:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditFilter"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelImage">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ListViewChange</class>
   <extends>QListView</extends>
   <header>ListViewChange.h</header>
   <slots>
    <signal>CurrentChanged(int)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BrowseSimulations</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BrowseSimulations</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listViewSimulation</sender>
   <signal>CurrentChanged(int)</signal>
   <receiver>BrowseSimulations</receiver>
   <slot>CurrentChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>CurrentChanged(int)</slot>
 </slots>
</ui>
